src/components/OrgFile/components/PropertyListEditorModal/index.js

```diff
@@ -85,6 +85,12 @@ export default class PropertyListEditorModal extends PureComponent {
       <Drawer onClose={onClose}>
         <h2 className="drawer-modal__title">Edit property list</h2>
 
+        <datalist id="datalist-property-names">
+          <option value="blocks"/>
+          <option value="depends"/>
+          <option value="assignee"/>
```

Comment:
@munen could you give me a hint where to get a combined list of all properties of the file from?

In other words, where is the modal called from and where are the arguments passed? Sorry for that beginner question...

Reply:
@schoettl One of the benefits of the used stack (React, Redux) is that you've got great tooling to work on these questions! Both of them have developer tools that you can install to your browser. Using them, you can see that the `Header` component receives this info as props:

![image](https://user-images.githubusercontent.com/505721/71323483-0728c300-24d4-11ea-9231-820d9eb21647.png)

[Here's](https://github.com/200ok-ch/organice/blob/master/src/components/OrgFile/components/HeaderList/index.js#L127) where that rendering is happening and the props are passed.

If you go further down the rabbit hole, you'll see that the `HeaderList` component get's this information itself through props [here](https://github.com/200ok-ch/organice/blob/master/src/components/OrgFile/components/HeaderList/index.js#L43-L55) and [here](https://github.com/200ok-ch/organice/blob/master/src/components/OrgFile/components/HeaderList/index.js#L143). 

organice implements the redux connect pattern here with the function `mapStateToProps`. This pattern is documented here: https://react-redux.js.org/using-react-redux/connect-mapstate